### ISSUE TITLE
Disabled styles do not match USWDS

### ISSUE THREAD

**Issue by rachidatecs** [cisagov/manage.get.gov#2225]
### Current Behavior

Our disabled input styles have been customized and do not look like USWDS's.

NOTE: Suggest time boxing investigation as to why USWDS's disabled styles are not being pulled (see additional context below) to a few hours. If no progress, just edit the existing scss in forms.scss and determine whether we make a ticket to investigate the devops bug more thoroughly.

### Expected Behavior

Disabled inputs should have the same background and text color as USWDS's disabled inputs.

### Steps to Reproduce

1. Navigate to a form with a disabled text input (also check disabled select)

### Additional Context

Review of PR #

> Noting that our disabled background color for the text input is not matching what [USWDS uses](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cm-create-error-stories/?path=/story/components-form-inputs-text-input--disabled). Rachid, I know you had adjusted this a while back? Was it due to contrast errors?
> 
> USWDS's storybook input ![Screenshot 2024-05-22 at 5 10 13 PM](https://private-user-images.githubusercontent.com/107440934/332976109-188b8de3-80fb-4a6c-93dd-bbca6caa2028.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0ODE2MzYsIm5iZiI6MTcxNjQ4MTMzNiwicGF0aCI6Ii8xMDc0NDA5MzQvMzMyOTc2MTA5LTE4OGI4ZGUzLTgwZmItNGE2Yy05M2RkLWJiY2E2Y2FhMjAyOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyM1QxNjIyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNDU4YTQxMWNiZDEwYTA2ZWY1YjExOTEyYThmNzA5N2M2NzVlN2UzMzkxMzEwYzA2NjhkOTJlYTFhMzcxY2Q3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.W0GLcVdOipLbF32osGSYnalOGMTJx6KelRxw1apuFyg)
> 
> Our input ![Screenshot 2024-05-22 at 5 10 23 PM](https://private-user-images.githubusercontent.com/107440934/332976101-2346bcb6-47ba-4b20-ac89-be238e622563.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0ODE2MzYsIm5iZiI6MTcxNjQ4MTMzNiwicGF0aCI6Ii8xMDc0NDA5MzQvMzMyOTc2MTAxLTIzNDZiY2I2LTQ3YmEtNGIyMC1hYzg5LWJlMjM4ZTYyMjU2My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyM1QxNjIyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYzA4Yjk3NTk2YTJjYTA4MmQ5ZjRmMzQ4MzEwNTk1OWMyZDllZDZlYzYyOWMzYjVlZmNkNTdiMmRkMzA3N2UyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.kP6Yb7C5oK836Zex0ocF7JWDlmcELPxEnMKWMQ55dyU)

I believe @zandercymatics wrote the relevant code:

```
@media (prefers-color-scheme: light) {
  .usa-input:disabled, .usa-select:disabled, .usa-textarea:disabled {
    background-color: #eeeeee;
    color: #666666;
  }
}

@media (prefers-color-scheme: dark) {
  .usa-input:disabled, .usa-select:disabled, .usa-textarea:disabled {
    background-color: var(--body-fg);
    color: var(--close-button-hover-bg);
  }
}
```

It looks like this had to be done because the usa-input disabled classes you see on USWDS are not being compiled in our project, for some reason. I recommend creating a bug ticket to look into this and revise (if needed) as this is a pre-existing issue that does not have to hold up this PR.

**Comment by PaulKuykendall** [comment]
This doesn't match the style guide, but we're **not sure** that it's not Sec.508 compliant.
